<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationPage</class>
 <widget class="QWizardPage" name="ConfigurationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="page_layout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scroll_area_">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <widget class="QWidget" name="content_widget_">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>580</width>
        <height>600</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="content_layout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QGroupBox" name="confidence_group_">
         <property name="title">
          <string>Confidence Threshold</string>
         </property>
         <layout class="QVBoxLayout" name="confidence_layout">
          <item>
           <layout class="QHBoxLayout" name="confidence_slider_layout">
            <item>
             <widget class="QLabel" name="confidence_min_label_">
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="confidence_slider_">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="confidence_max_label_">
              <property name="text">
               <string>1.0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="confidence_value_label_">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>0.50</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="confidence_help_">
            <property name="styleSheet">
             <string notr="true">color: #666;</string>
            </property>
            <property name="text">
             <string>Higher = fewer detections, more precise
Lower = more detections, may include false positives</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="nms_group_">
         <property name="title">
          <string>NMS IoU Threshold</string>
         </property>
         <layout class="QVBoxLayout" name="nms_layout">
          <item>
           <layout class="QHBoxLayout" name="nms_slider_layout">
            <item>
             <widget class="QLabel" name="nms_min_label_">
              <property name="text">
               <string>0.0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="nms_slider_">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="nms_max_label_">
              <property name="text">
               <string>1.0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="nms_value_label_">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font-weight: bold;</string>
              </property>
              <property name="text">
               <string>0.50</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="nms_help_">
            <property name="styleSheet">
             <string notr="true">color: #666;</string>
            </property>
            <property name="text">
             <string>Controls overlap filtering (0.3-0.7 typical)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="device_group_">
         <property name="title">
          <string>Device</string>
         </property>
         <layout class="QVBoxLayout" name="device_layout">
          <item>
           <widget class="QRadioButton" name="device_auto_radio_">
            <property name="text">
             <string>Auto (GPU if available)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="device_cpu_radio_">
            <property name="text">
             <string>Force CPU</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="device_gpu_radio_">
            <property name="text">
             <string>Force GPU (CUDA)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="device_mps_radio_">
            <property name="text">
             <string>Force Apple Silicon (MPS)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="advanced_group_">
         <property name="title">
          <string>Advanced Settings</string>
         </property>
         <layout class="QFormLayout" name="advanced_layout">
          <item row="0" column="0">
           <widget class="QLabel" name="min_size_label_">
            <property name="text">
             <string>Min object size:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="min_size_spin_">
            <property name="suffix">
             <string> pixels</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>32</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="max_detections_label_">
            <property name="text">
             <string>Max detections:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="max_detections_spin_">
            <property name="suffix">
             <string> per image</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="image_size_label_">
            <property name="text">
             <string>Image size (resize):</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="image_size_spin_">
            <property name="suffix">
             <string> px (shorter edge)</string>
            </property>
            <property name="minimum">
             <number>320</number>
            </property>
            <property name="maximum">
             <number>2048</number>
            </property>
            <property name="singleStep">
             <number>32</number>
            </property>
            <property name="value">
             <number>800</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
